query/src/sessions/session_mgr.rs

```diff
@@ -232,14 +233,15 @@ impl SessionManager {
             tracing::info!("Will shutdown forcefully.");
             active_sessions
                 .read()
+                .await
                 .values()
                 .for_each(Session::force_kill_session);
         }
     }
 
     pub fn processes_info(self: &Arc<Self>) -> Vec<ProcessInfo> {
-        self.active_sessions
-            .read()
+        let sessions = futures::executor::block_on(self.active_sessions.read());
```

Comment:
Should avoid using `block_on`, if it possible let's make the function to `async`.

Reply:
https://github.com/datafuselabs/databend/pull/4361#discussion_r822347236

Reply:
Solved, thanks @zhang2014